esp/src/package.json

```diff
@@ -34,69 +34,68 @@
   },
   "main": "src/stub.js",
   "dependencies": {
-    "@fluentui/react": "^8.57.0",
+    "@fluentui/react": "8.61.0",
```

Comment:
@GordonSmith I also pulled in the changes to the package & lock files from #15882 to avoid any conflicts, depending on which one gets merged first

Reply:
(In general I wouldn't go out of my way to do this - unless it was making my life easier in which case its fine)